packages/create-cloudflare/templates/angular/c3.ts

```diff
@@ -6,50 +6,23 @@ import { spinner } from "@cloudflare/cli/interactive";
 import { installPackages, runFrameworkGenerator } from "helpers/command";
 import { compatDateFlag, readFile, readJSON, writeFile } from "helpers/files";
 import { detectPackageManager } from "helpers/packages";
-import type { FrameworkConfig, PagesGeneratorContext } from "types";
+import type { C3Context } from "types";
+import { TemplateConfig } from "../../src/templateMap";
 
 const { npm } = detectPackageManager();
 
-const generate = async (ctx: PagesGeneratorContext) => {
+const generate = async (ctx: C3Context) => {
 	await runFrameworkGenerator(ctx, [ctx.project.name, "--ssr"]);
-
 	logRaw("");
 };
 
-const configure = async (ctx: PagesGeneratorContext) => {
-	process.chdir(ctx.project.path);
+const configure = async (ctx: C3Context) => {
 	updateAngularJson(ctx);
 	await updateAppCode();
 	await installCFWorker(ctx);
 };
 
-const config: FrameworkConfig = {
-	generate,
-	configure,
-	displayName: "Angular",
-	getPackageScripts: async () => ({
-		start: `${npm} run pages:build && wrangler pages dev dist/cloudflare ${await compatDateFlag()} --experimental-local`,
-		process: "node ./tools/copy-files.mjs && node ./tools/alter-polyfills.mjs",
-		"pages:build": `ng build && ${npm} run process`,
-		deploy: `${npm} run pages:build && wrangler pages deploy dist/cloudflare`,
-	}),
-	deployCommand: ["deploy"],
-	devCommand: ["start"],
-	testFlags: ["--style", "sass"],
-};
-export default config;
-
-async function installCFWorker(ctx: PagesGeneratorContext) {
-	const s = spinner();
-	s.start(`Adding Cloudflare Pages adapter code`);
-	await cp(
-		// eslint-disable-next-line no-restricted-globals
-		resolve(__dirname, "./angular/templates"),
-		resolve(ctx.project.path),
-		{ recursive: true, force: true }
-	);
-	s.stop(`${brandColor("copied")} ${dim("adapter code")}`);
-
+async function installCFWorker(ctx: C3Context) {
 	await installPackages(
 		["@cloudflare/workers-types", "@miniflare/tre@next", "wrangler@beta"],
```

Comment:
Not related to this PR but I think we probably can get rid of the `@next` and `@beta` here